teachertool/src/components/EvalResultDisplay.tsx

```diff
@@ -1,11 +1,39 @@
+import * as React from "react";
 import { useContext } from "react";
+import css from "./styling/EvalResultDisplay.module.scss";
 import { AppStateContext } from "../state/appStateContext";
 import { getCatalogCriteriaWithId } from "../state/helpers";
-import { CriteriaEvaluationResult } from "../types/criteria";
+import { CriteriaResultEntry } from "./CriteriaResultEntry";
+import { QRCodeSVG } from "qrcode.react"
 
-interface IProps {}
 
-export const EvalResultDisplay: React.FC<IProps> = ({}) => {
+const ResultsHeader: React.FC = () => {
+    const { state: teacherTool } = useContext(AppStateContext);
+
+    function getProjectLink(): string {
+        const inputText = teacherTool.projectMetadata?.inputText!;
+        const hasMakeCode = inputText?.indexOf("makecode") !== -1;
+        return hasMakeCode ? inputText : `https://makecode.com/${inputText}`;
+    }
+    // TODO: change the headers to be the correct thing for html (not h3, h4)
+    return (
+        <div className={css["header"]}>
+            <div className={css["rubric-name"]}>
+                <h2>{lf("{0}",teacherTool.rubric.name)}</h2>
+            </div>
+            <div className={css["project-details"]}>
+                <div className={css["project-text"]}>
+                    <h3>{lf("{0}", teacherTool?.projectMetadata?.name)}</h3>
```

Comment:
Also non-localizable.
```suggestion
                    <h3>{teacherTool?.projectMetadata?.name}</h3>
```